test/unit/constants.dart

```diff
@@ -21,7 +21,7 @@ final List<String> skippedDraft6FormatTestFiles = const [
 /// Commented out lines are NOT skipped.
 final List<String> skippedDraft7FormatTestFiles = const [
   // 'date.json',
-  'idn-email.json',
+  // 'idn-email.json',
```

Comment:
We actually pass most of these, not sure why the whole file was skipped :man_shrugging:

Reply:
If they mostly passing now, I think you can remove the name from this list.

Reply:
Sure, if we're skipping none of them anymore I could do that if @michaelcarter-wf agrees